### PR TITLE
Updates to switch to JARs, and corrections for Blogs series

### PR DESCRIPTION
**Pull request by nycnewman**

### CODE REVIEW

File: run-json-api.sh [hunk 1]
```diff
@@ -16,14 +16,25 @@ fi
 
 ./get-json-api-token.sh
 
-daml json-api --ledger-host ledger.$DOMAIN \
+#daml json-api --ledger-host ledger.$DOMAIN \
```
Comment (cocreature) [inline run-json-api.sh]: If you don’t use it, why not delete it?

File: run-sandbox.sh [hunk 2]
```diff
@@ -36,7 +37,22 @@ sleep 5
 # --auth-jwt-rs256-jwks=<url>
 # --auth-jwt-hs256-unsafe=<secret>
 
-daml sandbox \
+#daml sandbox \
```
Comment (cocreature) [inline run-sandbox.sh]: Same here, can we kill it?

**Review by cocreature — APPROVED** [review]
Thanks!

File: run-json-api.sh (hunk 1 above)
Comment (nycnewman) [reply run-json-api.sh]: fixed in commit

File: run-sandbox.sh (hunk 2 above)
Comment (nycnewman) [reply run-sandbox.sh]: fixed in commit

**Review by kareem-DA — COMMENTED** [review]
Looks good

**Pull request merged**